R/controls.R

```diff
@@ -46,7 +30,9 @@ ionrangesliderLibs <- function() {
       script = "js/ion.rangeSlider.min.js",
       # ion.rangeSlider also needs normalize.css, which is already included in
       # Bootstrap.
-      stylesheet = c("css/ion.rangeSlider.css",
+      stylesheet = c(
+        "css/normalize.css",
```

Comment:
Was this added because `filter_slider()` looks bad without it when Bootstrap isn't present? It generally seems like a bad idea for the same reasoning that importing all of Bootstrap also leads "browser resets" leads to conflicts with other CSS frameworks that want control over the resets.

Even if `filter_slider()` looks bad without this, let's not worry about it for now -- we'd also like to update ionrangeslider to match shiny's current version which I think will solve the normalize/reset issue